repository-hpi/src/main/java/com/nirima/jenkins/SelectionTypeProject.java

```diff
@@ -105,7 +105,7 @@ private BuildableItemWithBuildWrappers getProject(final String project) {
                 Jenkins.getInstance().getAllItems(BuildableItemWithBuildWrappers.class),
                 new Predicate<BuildableItemWithBuildWrappers>() {
                     public boolean apply(BuildableItemWithBuildWrappers buildableItemWithBuildWrappers) {
-                        return buildableItemWithBuildWrappers.getName().equals(project);
+                        return buildableItemWithBuildWrappers.getFullName().equals(project);
                     }
                 });
         return item;
```

Comment:
That may be true, but I tried to make smallest changes possible. There are many other places in this plugin that can be improved.